include/zephyr/drivers/usb_c/usbc_pd.h

```diff
@@ -53,6 +53,93 @@ extern "C" {
  */
 #define PD_MAX_EXTENDED_MSG_CHUNK_LEN    26
 
+/**
+ * @name USB PD 3.1 Rev 1.6, Table 6-70 Counter Parameters
+ * @{
+ */
+
+/**
+ * @brief The CapsCounter is used to count the number of Source_Capabilities
+ *	  Messages which have been sent by a Source at power up or after a
+ *	  Hard Reset.
+ *	  Parameter Name: CapsCounter
+ */
+#define PD_N_CAPS_COUNT 50
+
+/**
+ * @brief The HardResetCounter is used to retry the Hard Reset whenever there
+ *	  is no response from the remote device (see Section 6.6.6)
+ *	  Parameter Name: HardResetCounter
+ */
+#define PD_N_HARD_RESET_COUNT 2
+
+/** @} */
+
+/**
+ * @name USB PD 3.1 Rev 1.6, Table 6-68 Time Values
+ * @{
+ */
+
+/**
+ * @brief The NoResponseTimer is used by the Policy Engine in a Source to
+ *	  determine that its Port Partner is not responding after a Hard Reset.
+ *	  Parameter Name: NoResponseTimer
```

Comment:
Same issue here, the parameter names start with "t", but you've documented the "Timer" name from table 6-69

Reply:
done